code/datums/helper_datums/getrev.dm

```diff
@@ -21,13 +23,58 @@ var/global/datum/getrev/revdata = new()
 	if(testmerge.len)
 		world.log << commit
 		for(var/line in testmerge)
-			if(line)
-				world.log << "Test merge active of PR #[line]"
+			world.log << "Test merge active of PR #[line]"
 		world.log << "Based off master commit [parentcommit]"
 	else
 		world.log << parentcommit
 	world.log << "Current map - [MAP_NAME]" //can't think of anywhere better to put it
 
+/datum/getrev/proc/DownloadPRDetails()
+	for(var/line in testmerge)
+		//"This has got to be the ugliest hack I have ever done"
+		//warning, here be dragons
+		/*
+			                |  @___oo
+			      /\  /\   / (__,,,,|
+			     ) /^\) ^\/ _)
+			     )   /^\/   _)
+			     )   _ /  / _)
+			 /\  )/\/ ||  | )_)
+			<  >      |(,,) )__)
+			 ||      /    \)___)\
+			 | \____(      )___) )___
+			  \______(_______;;; __;;;
+		*/
+		var/url = "https://api.github.com/repositories/[config.githubrepoid]/pulls/[line].json"
+		var/temp_file = "prcheck[line].json"
+		var/command = "powershell -Command \"curl -O [temp_file] [url]\""

```

Comment:
I'm not even entertaining that idea until i fully understand why `world`.`Export()` won't work.